app/views/devise/registrations/new.html.slim

```diff
@@ -23,7 +23,11 @@ h2 = t('.title')
       .columns.small-4
         = f.input :birthdate, as: :string, input_html: {type: :date}
       .columns.small-6
-        = f.input :birthplace
+        .row
+          .columns.small-8
+            = f.input :birthplace
+          .columns.small-4
+            = f.input :birthplace_district, collection: District.all.invert
```

Comment:
In questa view ho fatto la chiamata diretta al model perchè il controller è di devise.